Scripts/GUI.cs

```diff
@@ -0,0 +1,16 @@
+using System;
+using Godot;
+
+public class GUI : Control
```

Comment:
Wanna name this something more specific?

Reply:
Like what? I was planning to make it just some helper class that can be used to access GUI elements from other Nodes.